<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AssetBrowserPreviewCharacterDlgFooter</class>
 <widget class="QWidget" name="AssetBrowserPreviewCharacterDlgFooter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>549</width>
    <height>50</height>
   </rect>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Lod Level:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="m_CBLodLevel">
     <property name="minimumSize">
      <size>
       <width>35</width>
       <height>0</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Ambient Multiplier:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSlider" name="m_sliderAmbient">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="tickPosition">
      <enum>QSlider::TicksBothSides</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="m_labelValue">
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>141</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
